src/zimscraperlib/download.py

```diff
@@ -27,11 +27,7 @@ def save_file(url, fpath, timeout=30, retries=0):
             with open(fpath, "wb") as fp:
                 fp.write(resp.content)
             return resp.headers
-        except (
```

Comment:
you should change you tests to test for all three exceptions (ans thus use case for those) instead of catching all exceptions

Reply:
Okay